index.html

```diff
@@ -14,6 +15,63 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container">
+
+    <!-- NAV BAR -->
+    <div class="nav-bar">
+      <a>Home</a>
+      <a>About</a>
+      <a>Portfolio</a>
+      <a>Contact</a>
+    </div>
+
+    <h1>"Mr NonImposter Syndrome"</h1>
+
+    <!-- INTRO -->
+    <div class="intro">
+      <img src="images/images.jpeg" alt="1337 h4x0r">
```

Comment:
I'd consider giving the `images.jpeg` file a more meaningful name; it's not super important, but it definitely confuses me a bit when I see an image called `images`.